apps/balancer-tools/src/components/SearchPoolForm.tsx

```diff
@@ -172,7 +172,12 @@ export function SearchPoolForm({
                       }}
                     >
                       <div className="flex w-full flex-col items-start">
-                        <span>{pool.symbol}</span>
+                        <span>
+                          {poolTypeFilter?.[0] == "FX" &&
+                          poolTypeFilter?.length == 1
+                            ? pool.name
+                            : pool.symbol}
+                        </span>
```

Comment:
can we add a comment explaining why to do this here?

Reply:
Yes. It is bc FX pools have "BPT" as the symbol for all of them.

Reply:
<img width="284" alt="image" src="https://github.com/bleu-studio/balancer-tools/assets/55461956/1f0447bc-e285-471a-ab8c-f217d4c2d31f">